src/EnergyPlus/PlantCondLoopOperation.cc

```diff
@@ -3175,18 +3175,19 @@ namespace PlantCondLoopOperation {
 						UniqueIDName = PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Name;
 						ActuatorType = "On/Off Supervisory";
 						SetupEMSActuator( ActuatorName, UniqueIDName, ActuatorType, Units, PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).EMSCtrlOverrideOn, PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).EMSCtrlOverrideValue );
-					} else if ( LoopSideNum == DemandSide ) {
+						for ( CompNum = 1; CompNum <= PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).TotalComponents; ++CompNum ) {
+							ActuatorName = "Plant Component " + ccSimPlantEquipTypes( PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).TypeOf_Num );
+							UniqueIDName = PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).Name;
+							ActuatorType = "On/Off Supervisory";
+							SetupEMSActuator( ActuatorName, UniqueIDName, ActuatorType, "[W]", PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).EMSLoadOverrideOn, PlantLoop( LoopNum ).LoopSide( LoopSideNum ).Branch( BranchNum ).Comp( CompNum ).EMSLoadOverrideValue );
+						}
+					}
+					else if ( LoopSideNum == DemandSide ) {
```

Comment:
Based on the description of this actuator, it seems there should be only one instance per component, not per plant connection:
"Each plant (and condenser) loop has a series of actuators called “Plant Component *” that are available with the control type “On/Off Supervisory.” These are available for each individual component on a loop. Setting the value of this actuator to 1.0 directs the component to run normally based on other controls. Setting the value of this actuator to 0.0 directs the component to shut down regardless of what other controls indicate."

So, the only problem with this fix would be if a particular component never appears on a supply side, and could still be a problem is it appears on two supply sides.  Next question is where/how these actuators get applied.  That my give a clue as to how they should be set up.